Process description quality management takeover from old process repository (#21)
eclipse-score/process_description: process/process_areas/quality_management/quality_roles.rst

..
   # *******************************************************************************
   # Copyright (c) 2025 Contributors to the Eclipse Foundation
   #
   # See the NOTICE file(s) distributed with this work for additional
   # information regarding copyright ownership.
   #
   # This program and the accompanying materials are made available under the
   # terms of the Apache License Version 2.0 which is available at
   # https://www.apache.org/licenses/LICENSE-2.0
   #
   # SPDX-License-Identifier: Apache-2.0
   # *******************************************************************************

> Roles

Reviewer: Out of curiosity, why do we extract the quality manager role from the normal role description? (Just wanted to know, as I may have missed the discussion)

#####

.. role:: Quality Manager
   :id: rl__quality_manager
   :status: valid

>    The quality managers shall be responsible for the planning and coordination of the quality activities, i.e. the quality management. He/She shall lead and monitor the quality relevant activities of the project.

Reviewer: ```suggestion
   The quality managers shall be responsible for the planning and coordination of the quality activities, i.e. the quality management. They shall lead and monitor the quality relevant activities of the project.
```
They or Their is often used as a neutral form as using of "he/she" will exclude people.

   Required skills

>    * Degree: Master degree in quality management, or similar technical/economical degree, or comparable work experience

Reviewer 2: software quality management

Author: Corrected it

>    * Degree: Master degree in software quality management, or similar technical/economical degree, or comparable work experience

Reviewer: I know about quality management degrees, and I know about software quality management courses as part of a MSc or BSc study, but are there really full time study programs on software quality management? And I would judge that an "computer science" degree is not a similar degree. Maybe this statement can be a bit relaxed as the things below already set the scene.

>    * Solid understanding of quality management

Reviewer 2: software quality management

Author: Corrected it

   * Knowledge in project management
   * Deep understanding of quality criteria and the correlating methods and procedures to achieve and verify them
   * Technical know-how of embedded systems
   * Preferred Training: ASPICE 4.0 (see training path, https://intacs.info/index.php/certification-center) at least Expert or Professional Assessor

   Knowledge of standards

   * Deep knowledge of ASPICE PAM 4.0
   * Basic knowledge of ISO 26262
   * Basic knowledge of ISOSAE 21434

   Experience

   * 2 years of experience in software quality management
   * Experience in managing projects

   Responsibility

   * Creating/maintain Quality Management Plan
   * Verify/approve Platform release
   * Execute Platform Process Audit

>    * Execute Feature Process conformance checks

Reviewer 2: Propose Execute Feature Contribution Conformance Checks

   * Execute Feature Work Product Reviews
   * Consult and execute Quality Trainings
   * Monitoring/improving of Quality activities

   Authority

>    * Escalation of planning topics to the project manager

Author: Changed to project leader acc. to PMP
